app/views/common/ExplanationScreen.tsx

```diff
@@ -136,14 +136,15 @@ const ExplanationScreen = ({
           style={primaryButtonStyles}
           textStyle={primaryButtonTextStyles}
         />
-        {explanationScreenContent.secondaryButtonLabel && (
-          <Button
-            label={explanationScreenContent.secondaryButtonLabel}
-            onPress={explanationScreenActions.secondaryButtonOnPress}
-            style={secondaryButtonStyles}
-            textStyle={secondaryButtonTextStyles}
-          />
-        )}
+        {explanationScreenActions.secondaryButtonOnPress &&
```

Comment:
@johnschoeman I'm going to merge this in because it's been open for a while, but I'm curious if there is a better way to do this check. I tried to pull it out into a constant but the check stopped working.